spec/ddtrace/contrib/rails/support/deprecation.rb

```diff
@@ -0,0 +1,8 @@
+# Raises error when a deprecation would be emitted
+def raise_on_rails_deprecation!
+  # DEV: In Rails 6.1 `ActiveSupport::Deprecation.disallowed_warnings`
+  # DEV: was introduced that allows fine grain configuration
```

Comment:
I don't think we need a `DEV` per line; just `DEV` on the first line and matching indentation should be sufficient.

Reply:
Done!